examples/player/player_wrapper.rs

```diff
@@ -14,14 +18,45 @@ use std::sync::{Arc, Mutex};
 use std::thread::Builder;
 
 pub struct PlayerWrapper {
-    player: Arc<Mutex<Box<Player>>>,
+    player: Arc<Mutex<Box<dyn Player>>>,
     shutdown: Arc<AtomicBool>,
+    use_gl: bool,
 }
 
 impl PlayerWrapper {
-    pub fn new(path: &Path) -> Self {
+    #[cfg(target_os = "linux")]
+    fn set_gl_params(player: Arc<Mutex<Box<dyn Player>>>, context: &glutin::Context) -> bool {
```

Comment:
Instead of returning `bool`, isn't this a case of failure if you return `false`? If so, some kind of `Result<(), _>` would seem nicer here.